frontend/src/global_styles/openproject/_mixins.sass

```diff
@@ -361,3 +361,7 @@ $scrollbar-size: 10px
     display: table
   &:after
     clear: both
+
+@mixin table-header-styles
```

Comment:
The name is a bit misleading as these are not our default table header styles, but rather those of tables inside an editor content. We already have a prefix for these kind of content called `op-uc`. Please use that here as well